content/docs/tasks/traffic-management/egress-gateway.md

```diff
@@ -126,6 +154,28 @@ the traffic through the egress gateway:
         EOF
     ```
 
+    In addition, if you have [mutual TLS Authentication](/docs/tasks/security/mutual-tls/) enabled in Istio, you must
+    create a destination rule to enable mutual TLS between the sidecar proxies and the egress gateway:
```

Comment:
This could use more explanation too. What is the default tls for incoming requests to egress gateway? I would think it should be ISTIO_MUTUAL if mTLS is globally enabled.

Reply:
@frankbu Good point, it actually is not needed, I was confused. Removing the destination rules for the egress gateway.